### PR TITLE
Add patent-searcher to showcase

### PR DESCRIPTION
**Pull request by yotamberk**
add #2854

### CODE REVIEW

**Comment by yotamberk** [comment]
done

**Comment by yotamberk** [comment]
@mojoaxel can you please release this PR so we can merge it?

**Comment by yotamberk** [comment]
@wimrijnders can you please approve this? I really want to get this PR off

File: showcase/index.html [hunk 1]
```diff
@@ -334,22 +334,35 @@
...
+                <div class="descriptionHeader">
+                    <a href="https://gbrault.github.io/railroad-diagrams/live/live.html">Timeline in PowerHA</a>
+                </div>
+                <div class="descriptionContent">
+                  PowerHA is a disaster recovery project where we developed a graphical interface for the command line system
+                  out clients has been using. Timeline is a graphical representation of all the events occurring in
+                  the environment over certain period of time.
+                </div>
+            </div>
+        </div>
```
Comment (wimrijnders) [inline showcase/index.html]: End tag does not line up with start tag.

File: showcase/index.html [hunk 2]
```diff
@@ -334,22 +334,35 @@
...
+        <div class="col-md-12 col-lg-6">
```
Comment (wimrijnders) [inline showcase/index.html]: Tags of this block do not align with previous block.

**Comment by yotamberk** [comment]
@wimrijnders done

**Pull request closed without merging**